gdtf.py

```diff
@@ -663,7 +665,6 @@ def get_axis(attribute):
             objs["2d_symbol"] = obj
             obj.show_in_front = True
             obj.active_material.grease_pencil.show_stroke = True
-            obj.data.pixel_factor = 2
```

Comment:
What is their plan for 4.3 and up, do you know? This was to make the lines thicker...

Reply:
@vanous I do not know the plan, but here is a changelog for 4.3
https://docs.blender.org/api/4.3/change_log.html